Diagnostic/RequiredExecutablesCheck.php

```diff
@@ -65,6 +65,6 @@ public function execute()
 
     private function isNohupPresent()
```

Comment:
Also you can change this method access to public and use the same one in Tasks.php
```suggestion
    public function isNohupPresent()
```

Reply:
@riczorn You changed the identifier back to private

Reply:
But you had changed it and I didn't touch it! I added a commit with it public again do you see it now?

Reply:
I can see it now